content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -444,6 +444,10 @@ kubectl get -o yaml job job-backoff-limit-per-index-example
       type: Failed
 ```
 
+Note that, since v1.31, you will also observe in the status the `FailureTarget`
```

Comment:
IIIUC, the Hugo shortcode would be better as you can see here: https://kubernetes.io/docs/contribute/style/style-guide/#shortcodes

Reply:
`{{< skew currentVersion >}}`